src/server/Document/Renderers/ComponentTrackingTemplate.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import processClientDeviceAndSendStaticBeacon from '#src/server/utilities/staticATITracking';
+import clickTracking from '#src/server/utilities/staticATITracking/clickTracking';
+import isOperaProxy from '#app/lib/utilities/isOperaProxy';
+import viewTracking from '#src/server/utilities/staticATITracking/viewTracking';
+
+const trackingScripts = ({
+  trackOnlyStaticCanonical = false,
+  trackComponentViews = true,
```

Comment:
I wonder if we should add a JSDoc to document these parameters and what they mean?